setup/install/providers/aws/aws-ec2.md

```diff
@@ -25,10 +25,13 @@ Use this option to deploy Spinnaker, if you are familar with deployment using [A
 1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
 2. Download [the template](https://d3079gxvs8ayeg.cloudfront.net/templates/managing.yaml) locally to your workstation.
 
-    2.a (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
-3. Creating the CloudFormation Stack
+    2.a Search for ‘SpinnakerInstanceProfileArn’ and comment out the line.
```

Comment:
Why does this line get commented out?  Is this assuming Spinnaker is deployed on EKS?

Reply:
This is asked to comment because as this Arn is restricting Clouddriver to load AWS account as clouddriver couldnt authenticate AWS Credentials. I have tested these on EKS, AKS, GKE and Vanilla K8s Cluster. Worked Perfectly fine

Reply:
Can you provide the error message you were seeing?  I'm not understanding why commenting out an output value in the CloudFormation stack would prevent CloudDriver from loading the account.

Reply:
Apologies if i had caused any confusion here, when i mean comment out. I actually want the 'SpinnakerInstanceProfileArn' to be commented.. I dont have the exception handy right now, i will have to do the setup again.. Need sometime for that

Reply:
I just tried, I believe it's probably this error, which is a bug in the template.
```
An error occurred (ValidationError) when calling the CreateChangeSet operation: Unresolved resource dependencies [SpinnakerInstanceProfile] in the Outputs block of the template
```

@dorbin I know at some point, the template source was in this github repo, and at some point it was taken out and replaced by a CloudFront link (https://github.com/spinnaker/spinnaker.github.io/pull/926).  I'm not sure what the reasoning was, but ideally the community would be able to own and contribute to the template (and fix bugs like this).  What do you think of moving it back into this repo?  Would it go in the "downloads" folder?

Reply:
Yeah, I don't know why those templates were deleted from here, and whether/where they were moved to.

@anshrma and @edwinavalos , I don't know if either of you remember the PR Clare has linked (#926 ), but feel free to chime in.

Failing that, as far as I'm concerned if it fixes the problem to add them back where they were, and doesn't break whatever Anuj and Edwin were trying to do in that PR, please do put them back in. I don't have an opinion on whether they should go back where they were or in the `downloads` folder; you can probably assess better than I can which is the safer option. 

@plumpy FYI

Also, @brian-armory and @aimeeu FYI, w/r/t the migration to the new repo.

Reply:
The template bug is now fixed as https://github.com/spinnaker/spinnaker.github.io/pull/2020, so this change can be reverted @anooptejt